app/src/main/java/com/hover/stax/ui/theme/Shape.kt

```diff
@@ -0,0 +1,11 @@
+package com.hover.stax.ui.theme
+
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Shapes
+import androidx.compose.ui.unit.dp
+
+val Shapes = Shapes(
```

Comment:
Maybe call this RoundedCornerShapes?

Reply:
This can be extended to include other shapes, not just the ones for rounded corners.